src/main/java/com/shopify/model/fulfillmentOrderApi/ShopifyDestination.java

```diff
@@ -0,0 +1,122 @@
+package com.shopify.model.fulfillmentOrderApi;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ShopifyDestination {
+
+	private String id;
```

Comment:
Did we need all of these fields? or did you just add them for response. I wonder if the fulfillment order object update should look different and only be a subset of fields like probably location id? I'm good with leaving as is, moreso just curoius.

Reply:
did not use any of this fields so far, but opted to have them since they might have use in the future...